build/docker/package.sh

```diff
@@ -5,7 +5,9 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 VERSION=1.23.0
 
 mkdir -p $DIR/../../deploy/linux
-cp $DIR/../../integrations.json $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/Debug/x64/
+for target in integrations.json defaults.env LICENSE NOTICE ; do
+    cp $DIR/../../$target $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/Debug/x64/
+done
```

Comment:
is it some additional fix?

Reply:
This is something that was changes in signalfx repo. I thought that was a good thing to have these files copied, so I took it :)